pdp/ast/yast/target.go

```diff
@@ -27,6 +27,13 @@ func (ctx context) getAdjustedArguments(v interface{}, val pdp.Expression, attr
 
 		return a, attr, nil
 
+	case pdp.PIPSelector:
```

Comment:
I'm going to remove ability to use selectors in target as set of targets from the same level policy sets, policies or rules is something which can be automatically indexed to speedup policy selection. The indexing isn't generally possible for expressions containing selectors (especially when network access is needed).